zproject/prod_settings_template.py

```diff
@@ -44,7 +44,7 @@
 ## representing the host/domain names that your users can enter in
 ## their browsers to access Zulip.  This is a security measure; for
 ## details, see the Django documentation:
-## https://docs.djangoproject.com/en/2.2/ref/settings/#allowed-hosts
+## https://docs.djangoproject.com/en/3.2/ref/settings/#allowed-hosts
 ##
 ## Zulip automatically adds to this list 'localhost', '127.0.0.1', and
```

Comment:
I'm not sure if this is of any importance, but I noticed that https://docs.djangoproject.com/en/2.2/ref/settings/#allowed-hosts has the line: 
`When DEBUG is True and ALLOWED_HOSTS is empty, the host is validated against ['localhost', '127.0.0.1', '[::1]'].`
and https://docs.djangoproject.com/en/3.2/ref/settings/#allowed-hosts has the line:
`When DEBUG is True and ALLOWED_HOSTS is empty, the host is validated against ['.localhost', '127.0.0.1', '[::1]'].`

Reply:
I think it is very possible we should be including `[::1]` in the `zproject/computed_settings.py` line that does this.  Can you start a thread on #backend about this question?  That's the right place to discuss.